UnitsNet/UnitDefinitions/MagneticFlux.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "MagneticFlux",
+  "BaseUnit": "Weber",
+  "XmlDoc": "In physics, specifically electromagnetism, the magnetic flux through a surface is the surface integral of the normal component of the magnetic field B passing through that surface.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Magnetic_flux",
+  "Units": [
+    {
+      "SingularName": "Weber",
+      "PluralName": "Webers",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
```

Comment:
Any prefix units to add? 
Add any of these units?
https://www.translatorscafe.com/unit-converter/en/magnetic-flux/1-10/weber-tesla%20meter%C2%B2/

Reply:
We listed in this PR only the units we're really using. We know that probably their relative physical quantities could be expanded with more definitions and prefixes but they're out of scope at the moment.
We are committed to adding them in future if their usage will become necessary.

Reply:
Fair enough, other units can be added later when needed.